src/domain/home/components/active_campaigns/campaign_card/CampaignCard.module.scss

```
@@ -0,0 +1,83 @@
+@import '../../../../../utils/variables';
+@import '../../../homeMixin';
+
+.campaignCard {
+    position: relative;
+    overflow: hidden;
+    object-fit: cover;
+    object-position: center;
+    margin: 0;
+    height: 100%;
+    width: 100%;
+    margin-top: 20px;
+
+    .gradient {
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: 50%;
+        background: linear-gradient(
+            180deg,
+            rgba(0, 0, 0, 0),
+            rgba(0, 0, 0, 0.6)
+        );
+    }
+
+    img {
+        object-position: center;
+        object-fit: cover;
+        width: 100%;
+        height: 100%;
+    }
+    .campaignTitle {
+        padding: 20px;
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        text-align: left;
+
+        h3 {
+            color: $white;
+            font-size: 30px;
+            margin: 0;
+            filter: drop-shadow(1px 1px 3px rgba($black, 0.8));
+        }
+    }
+    &::before {
+        z-index: 1;
+        opacity: 0;
+        content: 'View Campaign';
```

Note: I think we are trying to be consistent with the style of Captial first word and not capitalising the rest. (We aren't doing very well tho so there are probably many cases of this throughout the app and website)
```suggestion
        content: 'View campaign';
```